src/benchmarks/micro/libraries/System.Runtime.Numerics/Perf.BigInteger.cs

```diff
@@ -106,6 +106,44 @@ public IEnumerable<object> ModPowValues()
         public BigInteger ModPow(BigIntegers arguments)
             => BigInteger.ModPow(arguments.Left, arguments.Right, arguments.Other);
 
+        public IEnumerable<object> EqualsValues()
+        {
+            Random rnd = new Random(123456);
```

Comment:
thank you for using a constant seed 👍 (https://github.com/dotnet/performance/blob/main/docs/microbenchmark-design-guidelines.md#always-the-same-input-data)